### PR TITLE
f-checkout@0.17.0 - Add collection prop tests

### PR DESCRIPTION
**Pull request by siggerzz**
v0.17.0
------------------------------
*December 7, 2020*

### Added
- Component tests for 'Collection' prop variant.

### Changed
- Split component tests into separate spec files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "2.21.0",
+  "version": "2.21.1",
```
Comment (xander-marjoram) [inline package.json:3]: I don't think you need to bump this unless you're also bumping it in the changelog

Comment (siggerzz) [reply package.json:3]: An entry was made to the changelog 👍

Comment (xander-marjoram) [reply package.json:3]: But the version was not bumped, you can just replace `Latest (to be added to next release)` with `2.21.1` and that'll be groovy

Comment (siggerzz) [reply package.json:3]: done!

File: packages/f-checkout/test/specs/component/f-checkout-collection.component.spec.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import forEach from 'mocha-each';
+import DemoControls from '../../../test-utils/component-objects/demo-controls';
+import CheckoutComponent from '../../../test-utils/component-objects/f-checkout.component';
+
+describe('f-checkout "collection" component tests', () => {
+    forEach(['mobileNumber'])
+    .it.only('each fields error message should be displayed', field => {
```
Comment (xander-marjoram) [inline packages/f-checkout/test/specs/component/f-checkout-collection.component.spec.js]: Just checking that the `.only` here isn't preventing any of the other tests from being run?

**Review by xander-marjoram — DISMISSED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

**Pull request merged**